cstore_fdw.c

```diff
@@ -672,6 +681,55 @@ CStoreTable(Oid relationId)
 
 
 /*
+ * DistributedTable checks if the given relationId is the OID of a distributed table,
+ * which may also be a cstore_fdw table, but in that case COPY should be handled by
+ * Citus.
+ */
+static bool
+DistributedTable(Oid relationId)
+{
+	bool isDistributedTable = false;
+	Oid partitionOid = InvalidOid;
+	Relation heapRelation = NULL;
+	HeapScanDesc scanDesc = NULL;
+	const int scanKeyCount = 1;
+	ScanKeyData scanKey[scanKeyCount];
+	HeapTuple heapTuple = NULL;
+
```

Comment:
We usually have an empty line when transitioning from declarations to code (e.g. functions below)